### PR TITLE
TOUC-86 FE: On small screens dropdown appear in the top of screen wit…

### PR DESCRIPTION
**Pull request by pavel-stryber**
…h space

- fixed wrong View measuring on Android

### CODE REVIEW

File: src/components/Dropdown/index.tsx [hunk 1]
```diff
@@ -148,7 +149,8 @@ class Dropdown extends PureComponent<IDropdownProps, DropdownState> {
       // @ts-ignore
       dropdownMargins: {min: minMargin, max: maxMargin},
       animationDuration,
-      useNativeDriver,
+      useNativeDriver = false,
```
Comment (zaporozhetsAnton) [inline src/components/Dropdown/index.tsx:152]: do we need this assignment if we sat defaultProps.useNativeDriver = false on line 627?

Comment (pavel-stryber) [reply src/components/Dropdown/index.tsx:152]: typescript hints errors, that's why

File: src/components/Dropdown/index.tsx [hunk 2]
```diff
@@ -561,7 +554,7 @@ class Dropdown extends PureComponent<IDropdownProps, DropdownState> {
         <TouchableWithoutFeedback {...touchableProps}>
           <View pointerEvents="box-only">{this.renderBase(props)}</View>
         </TouchableWithoutFeedback>
-        <View ref={this.listContainer}>
+        <View ref={this.listContainer} nativeID="listContainer">
```
Comment (zaporozhetsAnton) [inline src/components/Dropdown/index.tsx:557]: Could you please explain what does nativeId prop do?

Comment (pavel-stryber) [reply src/components/Dropdown/index.tsx:557]: Android doesn't calculate properly x, y values in this.listContainer.current.measureInWindow method until you set some native property to the View.

**Pull request merged**